### ISSUE TITLE
Avoid writing tmp file?

### ISSUE THREAD

**Issue by igoyak** [YoungxHelsinki/automated-scripts#7]
https://github.com/YoungxHelsinki/automated-scripts/blob/3b8fd3813c73b96826b5693343e496797a13f9ee/Aalto_computer_status_checker/paniikki_jupyter_launcher.py#L231

I might still be misunderstanding the business logic, but it seems like it's unnecessary to write the tmp file. Have you tried feeding the code as stdin using the standard python method?
(this code is very quick and dirty, stolen from somewhere, just to make a point)

```
from subprocess import Popen, PIPE, STDOUT

p = Popen(['ssh', 'kosh', 'python'], stdout=PIPE, stdin=PIPE, stderr=STDOUT)    
CODE = b"""
import os
print("I am some python code")
print(os.uname())
"""
grep_stdout = p.communicate(input=CODE)[0]
print(grep_stdout.decode())

```

**Comment by SuperShinyEyes** [comment]
This is great. I didn't know that you could load the code in this way.

**Comment by SuperShinyEyes** [comment]
Fixed @ 97135732079d8600969d4ff02f6e339cadab8760
